cordova-lib/src/platforms/platforms.js

```diff
@@ -102,5 +102,10 @@ function getPlatformProject(platform, platformRootDir) {
 }
 
 module.exports = platforms;
-module.exports.getPlatformProject = getPlatformProject;
-module.exports.PlatformProjectAdapter = PlatformProjectAdapter;
+
```

Comment:
> used by a downstream project, so we need them.

Right - with this change they're still accessible (you should still be able to call them) - they're just not enumerated (in a `for in` loop for example).